proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Jesteś juz polaczony z tym serwerem!
+velocity.error.already-connected-proxy=Jesteś juz polaczony z tym serverem proxy!
+velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
+velocity.error.cant-connect=Nie można się połączyć z {0}: {1}
+velocity.error.connecting-server-error=Nie można się połączyć z {0}. Spróbuj ponownie później.
+velocity.error.connected-server-error=Twoje połączenie z {0} napotkało problem.
+velocity.error.internal-server-connection-error=Wystąpił wewnętrzny błąd połączenia z serwerem.
+velocity.error.logging-in-too-fast=Logujesz się zbyt szybko, spróbuj ponownie później.
+velocity.error.online-mode-only=Nie jesteś zalogowany to Minecrafta. Jeśli jesteś już zalogowany, sprubuj restartować twoj klient.
```

Comment:
spróbuj*
But for me "Na serwer mogą wejść osoby tylko z oryginalną kopią gry. Jeśli jesteś zalogowany na swoim koncie Minecraft, spróbuj zrestartować gre." sounds better.

Reply:
The original message ("You are not logged into your Minecraft account. If you are logged into your Minecraft account, try restarting your Minecraft client.") doesn't really say anything about having a paid copy of Minecraft. I am open to changing this if more people want it to be changed to include owning a copy.

Reply:
The implication is that you have a paid copy of Minecraft. That is precisely what we refer to when a user has online mode enabled.

Reply:
https://github.com/VelocityPowered/Velocity/pull/466/commits/1f81ff813696ae2f4fc9a09f9588b8ae1ae5a2b9